packages/mjml-head-font/README.md

```diff
@@ -1,11 +1,14 @@
 ## mj-font
 
-This tag allows you to import fonts if used in your MJML document
+This tag imports fonts for your use.
```

Comment:
I think the if used in your mjml document is important because it's not imported if not used once.

Reply:
Sure is. Changed to

This tag imports fonts.
The tag has effect only if the template uses the font, too.